common/src/main/java/com/wynntils/wc/custom/item/ItemStackTransformManager.java

```diff
@@ -78,14 +71,8 @@ protected void onInit(ImmutableList.Builder<Condition> conditions) {
         registerProperty(WynnItemMatchers::isMaterial, MaterialProperty::new);
     }
 
-    @Override
-    protected boolean onEnable() {
-        return (WebManager.isItemGuessesLoaded() || WebManager.tryLoadItemGuesses())
```

Comment:
Is this since these are permanently loaded now?

Reply:
Yes